### ISSUE TITLE
Internal Error On Certain Kits, Please Help ASAP

### ISSUE THREAD

**Issue by Will0aktree** [np98765/BattleKits#126]
Hello, I have been running a server with BattleKits for a while and I love the plugin. We had like 10 kits and everything worked fine. Then, when I made some more kits, I would get errors on certain kits like boomer, ranger, and wizard.

What Happens:
1. Only gives some items
2. Tells Me: An internal error occured while attemtping to perform this command
3. And I get this in the console (starts at bottom) http://pastebin.com/y6m2ki9z

Server Info:
1. Running CraftBukkit Beta 1.5.2 RO.1
2. Plugins Are:
TimTheEnchanter, PVPGunPlus, NoCheatPlus, WorldEdit, Vault, PermissionsEx, HealthBar, CombatTag, WorldGuard, PingMOTD, LinkSign, iConomy, KitMaster, CoreProtect, PVPCash, PvPTitles, ItemDropFix, ClearLag, DeathMessages, Votifier, GAListener, AutoAnnouncer, Essentials, BattleKits, CommandSigns, EssentialsSpawn, EssentialsChat
3. Im running the latest build of BattleKits

global.yml: http://pastebin.com/wsSanywE
kits.yml: http://pastebin.com/tAB5VQXp

Thank you for your time :)

**Comment by lol768** [comment]
Wrap enchantments in single quotes.......

**Comment by Will0aktree** [comment]
Can I get an example?

**Comment by lol768** [comment]
bootsEnchant: '1:0'

**Comment by lol768** [comment]
'Didn't work' is useless to me. Try again. Give me me your new configs and server log.

**Comment by Will0aktree** [comment]
It's the same error, and my other kits that work don't even have the single quotes on the enchants.

**Comment by lol768** [comment]
"my other kits that work don't even have the single quotes on the enchants."

Did you just completely ignore what I said to you earlier?

**Comment by Will0aktree** [comment]
God you are so damn rude, screw the plugin, I'll find another.

**Comment by lol768** [comment]
Please do, and good luck. 

If you can't edit a config file, might I recommend LiteKits? It's a plugin that uses your inventory to make the kit instead of a config file.

Have a nice day :smile: !

**Comment by Will0aktree** [comment]
As far as I'm concerned LiteKits isn't hooked to Vault.

**Comment by lol768** [comment]
Hi,

After looking back on how this ticket was handled, I see I did a terrible job of actually providing support.

Firstly, I apologise - I should have been more patient with getting this sorted than I originally was. Unfortunately, we've had loads of these tickets with the same issue and I'm starting to suspect that there's an issue with documentation somewhere. I've also had recent exams. Even so, these issues are nothing to do with your and I shouldn't have reacted the way I did. It frustrated me when you didn't fully explain the issue and it sounded like you'd only partially corrected the config.

If you'd still like this sorted, send me a pastebin of your latest config and I'll correct it for you. In terms of LiteKits, economy is something I feel is important too, and it'll probably end up in an extension at some point.

If you have any further queries, let me know and I'll do my best to try and sort it out. Apologies if I offended you at any point.
